apps/amo/tests/__init__.py

```diff
@@ -340,7 +340,7 @@ def assert3xx(self, response, expected_url, status_code=302,
                   target_status_code=200):
         """Asserts redirect and final redirect matches expected URL.
 
-        Similar to Django's `assertRedirects` but skips the final GET
+        Similar to Django's `assert3xx` but skips the final GET
         verification for speed.
 
         """
```

Comment:
I just created a 1.7 tracking bug and linked to your comment in https://github.com/mozilla/olympia/issues/905